.github/workflows/build.yml

```diff
@@ -34,6 +34,8 @@ jobs:
             platform: "i686"
           - policy: "manylinux_2_24"
             platform: "x86_64"
+          - policy: "musllinux_1_1"
+            platform: "x86_64"
 
```

Comment:
Any reason not to add i686 ?
IMHO, even if not used much, it brings parity with manylinux.